core-bundle/tests/Functional/RoutingTest.php

```diff
@@ -412,9 +412,9 @@ public function getAliasesWithLocale(): array
             ],
             'Renders the default page if the alias is empty' => [
```

Comment:
Renders the 404 page if the alias is empty

Reply:
Fixed in 5496681b13cec9843806184597d48623796eab1c